soc/wch/ch32v/qingke_v4c/vector.S

```diff
@@ -18,12 +18,14 @@ GTEXT(__initialize)
 
 SECTION_FUNC(vectors, ivt)
 	.option norvc
+	lui	x5, 0x8000
+	jalr 	x0, 0x8(x5)
```

Comment:
Why (and where) are we jumping here and discarding the return address? How is this related to enabling the whole flash on CH32V208?

Reply:
the full flash isnt accessible from booted-in zone 0x0, only 128KB, the full flash is available at 0x0800 0000 wit hthe first 128KB aliased to 0x0, this jumps to 0x0800 0008 to avoid a infintie loop.

Reply:
Makes sense - let's add that to a comment so that it's clear what we're doing here. Let's also change `jalr 	x0, 0x8(x5)` to pseudo `jr 0x8(x5)` to make the asm clearer